src/parse/kdbx3.rs

```diff
@@ -190,16 +190,15 @@ pub(crate) fn parse(data: &[u8], key_elements: &[Vec<u8>]) -> Result<Database, D
         uuid: Default::default(),
         name: "Root".to_owned(),
         children: Default::default(),
-        expires: Default::default(),
         times: Default::default(),
     };
 
     // Parse XML data blocks
     for block_buffer in xml_blocks {
-        let (block_group, _meta) = xml_parse::parse_xml_block(&block_buffer, &mut *inner_decryptor)
+        let kxml = crate::xml_db::parse::parse(&block_buffer, &mut *inner_decryptor)
```

Comment:
@sseemayer how about `raw_database`, or `database_content`?

Reply:
agreed and changed to `database_content`.